chore: Config travis

drazik opened the pull request:
I used the same configuration as cozy-notes.

REGISTRY_TOKEN is in password-store-team under
`registry/spaces/default/passwords.token`.

GITHUB_TOKEN has been generated with cozy-bot account.

drazik committed:
chore: Config travis

I used the same configuration as cozy-notes.

REGISTRY_TOKEN is in password-store-team under
`registry/spaces/default/passwords.token`.

GITHUB_TOKEN has been generated with cozy-bot account.

Crash-- commented:
We should not use the one from Notes since we want to change it. 

We should use downcloud to store our artefacts and not using a github branch to do that.

drazik commented:
Oh, I thought you did the changes we wanted to do already. I'm going to update the config to use downcloud, then

drazik committed:
fix(deps): Update cozy-bar to v7.12.2

yarn.lock has a lot of differences because it was regenerated from
scratch, due to errors `error https://github.com/cozy/minilog.git:
Extracting tar content of undefined failed, the file appears to be
corrupt: "Invalid tar header. Maybe the tar is corrupted or it needs to
be gunzipped?"`

drazik committed:
chore: Use downcloud to publish

drazik commented:
@Crash-- I did the change to use downcloud

Crash-- commented on .travis.yml:
```diff
@@ -1,60 +1,38 @@
+  - provider: script
+    repo: cozy/cozy-passwords
+    skip-cleanup: true
+    # deploy the build on a build branch and publish to the Cozy registry
```
```suggestion
    # deploy the build on a downcloud and publish to the Cozy registry
```

Crash-- reviewed (approved):
Amazing!

drazik committed:
chore: Remove unknown key in travis config

Travis outputs the following warning: `deploy: unknown key repo`

drazik committed:
chore: Use skip_cleanup instead of skip-cleanup

The key was automatically corrected by travis, but the good key is
skip_cleanup.
